### PR TITLE
Arthur add colours update

### PR DESCRIPTION
**Pull request by ArthurConmy**
Preview: 

[kl--trained--random_ablation--roc_edges.pdf](https://github.com/ArthurConmy/Automatic-Circuit-Discovery/files/11779900/kl--trained--random_ablation--roc_edges.pdf)

### CODE REVIEW

File: notebooks/plotly_roc_plot.py [hunk 1]
```diff
@@ -113,13 +130,18 @@
...
-                             [1e-16, color_list[0][1]],
+                             [0.01, color_list[0][1]],
                              *color_list[1:]]
 
     if methodof == "HISP" and args.hisp_yellow:
         colorscales[methodof][1][1] = "rgb(255, 255, 0)"
 
-colors = {k: pc.sample_colorscale(v, 0.01 if k == "HISP" and args.hisp_yellow else 0.2)[0] for k, v in colorscales.items()}
+custom_color_scales = {
+    ("HISP", args.hisp_yellow): 0.02,
```
Comment (rhaps0dy) [inline notebooks/plotly_roc_plot.py:140]: I think this should be `("HISP", True)`, but unsure

File: notebooks/plotly_roc_plot.py [hunk 2]
```diff
@@ -294,6 +316,11 @@ def normalize(x, x_min, x_max):
                 y_data = np.array(this_data[task_idx][metric_name][alg_idx][y_key])
                 scores = np.array(this_data[task_idx][metric_name][alg_idx]["score"])
 
+                if alg_idx.lower() in ["hisp", "16h"]:
+                    scores = np.array(this_data[task_idx][metric_name][alg_idx]["n_nodes"])
+                    for i in range(1, len(this_data[task_idx][metric_name][alg_idx]["n_nodes"])-1): # first and last broken I think
```
Comment (rhaps0dy) [inline notebooks/plotly_roc_plot.py:321]: the score is not the number of nodes, it's the sum of pruning scores for HISP and the lambda-regularizer for 16h.

File: notebooks/plotly_roc_plot.py [hunk 3]
```diff
@@ -97,9 +114,9 @@
 
 if args.hisp_yellow:
     colorscale_names = {
-        "ACDC": "Blues_r",
+        "ACDC": "Purp",
         "SP": "Greens_r",
-        "HISP": "pinkyl",
+        "HISP": "YlOrBr",
```
Comment (rhaps0dy) [inline notebooks/plotly_roc_plot.py:119]: I would like to standardize on "Darker is higher" or "Darker is lower" and that's not used here

**Comment by ArthurConmy** [comment]
Great, I merged https://github.com/ArthurConmy/Automatic-Circuit-Discovery/pull/62 which contains these changes

**Pull request closed without merging**